integrations/pagerduty/clients/pagerduty.py

```diff
@@ -191,3 +191,27 @@ async def update_oncall_users(
                 if user["escalation_policy"]["id"] == escalation_policy_id
             ]
         return services
+
+    async def get_incident_analytics(self, incident_id: str) -> dict[str, Any]:
+        logger.info(f"Fetching analytics for incident: {incident_id}")
+        url = f"{self.api_url}/analytics/raw/incidents/{incident_id}"
+
+        try:
+            response = await self.http_client.get(url)
+            response.raise_for_status()
+            data = response.json()
+            return data
+        except httpx.HTTPStatusError as e:
+            if e.response.status_code == 404:
+                logger.error(
+                    f"Incident {incident_id} analytics data was not found, skipping..."
+                )
+                return {}
```

Comment:
if we skip, is this actual an error log? we want to log an error if there is an actual problem so it won't be too noisy and only important error that needs to be checked will be logged as error